docs/tutorial/part06.rst

```diff
@@ -2,5 +2,649 @@
 How to serialize the data.
 ==========================
 
-.. todo::
-   Here will be the tutorial body.
\ No newline at end of file
+Why we need to serialize the data
+=================================
+
+The nodes that we are fetching so far are just instances of the model classes we
+defined(Entity, Officer, etc), we want to return JSON files to the fetch calls that are going to be

```

Comment:
Two options:

1 - '...the model classes we have defined such as Entity, Officer, etc. We want...' (Separating the sentences just a period after etc)
2 - '...the model classes we have defined such as Entity, Officer, etc., we want...' (Keeping a single sentence etc. followed by a comma)

Personally I like the first one...